announcements/v6.2.txt

```diff
@@ -8,7 +8,7 @@ and many other package repos, all found on GitHub. If you have time and
 if you wish to get involved, please take a look at the GitHub repos and
 find your favorite places to learn, fix, and enhance our world.
 
-The core repo is at https://github.com/plt/racket, and the
+The core repo is at https://github.com/racket/racket, and the
```

Comment:
I don't think this one should change.
The 6.2 announcement is at this point more of a "historical document", and so having it reflect things as they were then is the right thing, IMO.

Reply:
@stamourv fixed you comments